.github/workflows/build-size-comparison.yml

```
@@ -0,0 +1,138 @@
+        token: '${{ secrets.ACCESS_KEY }}'
+
+    - name: Checkout Premium Repo (Base Branch)
+      uses: actions/checkout@v4
+      with:
+        repository: 'bfintal/Stackable-Premium'
+        ref: 'v3'
+        path: 'base-branch/pro__premium_only'
+        token: '${{ secrets.ACCESS_KEY }}'
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)